### PR TITLE
UI enhancement of LoginViewController, SignUpViewController

### PR DESCRIPTION
**Pull request by ishaan1995**
Fixed LoginViewController and SignUpViewController UI, added keyboard dismiss functionality
Issue Mentioned - #28 

### Screen 1 (LoginViewController)
screen1-issue
### Screen 2 (SignUpViewController)
screen2-issue

### Screen 1 (LoginViewController)
screen1-fix
### Screen 2 (SignUpViewController)
screen2-fix

### CODE REVIEW

File: Susi/SignUpViewController.swift [hunk 1]
```diff
@@ -141,6 +167,19 @@ class SignUpViewController: UIViewController {
         
     }
     
+    //close keyboard on return click on keyboard
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        self.view.endEditing(true)
```
Comment (chashmeetsingh) [inline Susi/SignUpViewController.swift]: Call the dismissKeyboardOnTap() method here instead.

Comment (ishaan1995) [inline Susi/SignUpViewController.swift]: This has been resolved and squashed into this PR commit.

**Comment by ishaan1995** [comment]
This has been resolved and squashed into this PR commit.

**Comment by chashmeetsingh** [comment]
This PR can be merged, it's good to go.

**Comment by mariobehling** [comment]
Please check if issues with travis persists in later PRs. Thanks.

**Comment by chashmeetsingh** [comment]
Yes will do.

**Pull request merged**